genesyscloud/resource_genesyscloud_routing_queue.go

```diff
@@ -513,19 +525,10 @@ func ResourceRoutingQueue() *schema.Resource {
 }
 
 func createQueue(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	name := d.Get("name").(string)
-	divisionID := d.Get("division_id").(string)
-	description := d.Get("description").(string)
-	skillEvaluationMethod := d.Get("skill_evaluation_method").(string)
-	autoAnswerOnly := d.Get("auto_answer_only").(bool)
-	enableTranscription := d.Get("enable_transcription").(bool)
-	suppressInQueueCallRecording := d.Get("suppress_in_queue_call_recording").(bool)
```

Comment:
It looks like this isn't replaced with anything below. That's probably why you had to add this field to ImportStateVerifyIgnore in the tests